src/pcrf/pcrf-context.c

```diff
@@ -267,7 +267,7 @@ int pcrf_context_parse_config(void)
 }
 
 int pcrf_db_qos_data(
-        char *imsi_bcd, char *apn, ogs_session_data_t *session_data)
+        char *imsi_bcd, char *apn, int32_t charging_char, ogs_session_data_t *session_data)
```

Comment:
If ChargingChar is 16bit, did you think about passing a "uint16_t *" ptr and pass it NULL if none is to be used?

Reply:
I think having a int32 is simpler than passing a pointer.